addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -64,8 +73,34 @@ def _are_you_inside(self):
 
     @api.depends('user_ids.im_status')
     def _compute_available_operator_ids(self):
+        session_count_by_operator = {}
+        operator_in_call = self.env["res.partner"]
+        if self.concurent_session:
+            session_count_by_operator = dict(
+                self.env["discuss.channel"]._read_group(
+                    [
+                        ("livechat_channel_id", "!=", False),
+                        ("livechat_active", "=", True),
+                        ("last_interest_dt", ">", fields.Datetime.now() - timedelta(minutes=30)),
```

Comment:
Isn't there a "OR" missing? It should be "either not active or older than 30 minutes". If so, we should fix the tests as well.

Also, shouldn't it be "<="? It looks like we're looking for threads having interest in the last 30 minutes while it should be the opposite?